pkg/pillar/scripts/device-steps.sh

```diff
@@ -448,6 +448,13 @@ done
 $BINDIR/vaultmgr -c "$CURPART" runAsService &
 wait_for_touch vaultmgr
 
+# Start tpmmgr as a service
```

Comment:
Why is this started here instead of downstream from here (with all the other agents)?

Reply:
> Why is this started here instead of downstream from here (with all the other agents)?

The intent is to start TPM Manager only on systems where TPM is present, that is the reason it is not added to AGENTS1 list, and started explicitly here, upon checking for TPM presence.

Reply:
Fair enough @cshari-zededa